setup.py

```diff
@@ -16,8 +16,28 @@
 
 setup(name='haas',
       version='0.2rc2',
+      author='https://github.com/CCI-MOC/haas/network/members',
+      author_email = 'haas-dev-list@bu.edu',
       url='https://github.com/CCI-MOC/haas',
+      description='Hardware-as-a-Service or HaaS is a bare-metal isolation service that automates \
+allocation and management of non-virtualized compute resources',
+      long_description='HaaS allows mutually untrusting and incompatible bare metal services \
+to be deployed in a data center. By installing HaaS, data centers can easily \
+match change in demand by moving resources across pools thereby increasing overall \
+utilization and efficiency.',
+      license='Apache 2.0',
+      classifiers=['Development Status :: 4 - Beta',
+			'Intended Audience :: Developers :: System Administrators :: Science/Research', 
+			'Topic :: Cloud :: System :: Installation/Setup :: Systems Administration :: Clustering :: Utilities ',
```

Comment:
Ian is right - each classifier has to be a line from https://pypi.python.org/pypi?%3Aaction=list_classifiers